src/templates/Article.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
```

Comment:
@peterbaile Does Gatsby require you to import React when there's JSX in the file? Next doesn't